packages/snap/src/hedera/config.ts

```diff
@@ -6,6 +6,10 @@ const hederaChainIDs: Record<string, string> = {
   '0x129': 'previewnet',
 };
 
+const evmChainIDs: Record<string, string> = {
```

Comment:
any reason why this is under hedera? It's better if we move this to some other file because polygon is not hedera @dchagastelles